internal/infrastructure/rdb/place.go

```diff
@@ -572,6 +574,55 @@ func (p PlaceRepository) SaveGooglePlaceDetail(ctx context.Context, googlePlaceI
 	return nil
 }
 
+func (p PlaceRepository) SavePlacePhotos(ctx context.Context, userId string, placeId string, photoUrl string, width int, height int) error {
+	if err := runTransaction(ctx, p, func(ctx context.Context, tx *sql.Tx) error {
+		var ok bool
+		var err error
+		ok, err = generated.PlacePhotos(generated.PlacePhotoWhere.PhotoURL.EQ(photoUrl)).Exists(ctx, tx)
```

Comment:
[refactor] 変数のスコープは小さめに
```suggestion
		ok, err := generated.PlacePhotos(generated.PlacePhotoWhere.PhotoURL.EQ(photoUrl)).Exists(ctx, tx)
```

Reply:
[imo] あと、foundとかにするのはいかが？